sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -345,7 +345,7 @@ def extract_key_phrases(  # type: ignore
                 cls=key_phrases_result,
                 **kwargs
             )
```

Comment:
Would you mind updating in both clients everywhere where `cls=xxx_result` to `kwargs.pop("cls", xxx_result)` ? We need to give the user the option to use `cls`.